examples/convert_eeg_to_bids.py

```diff
@@ -82,20 +89,47 @@
 # MNE-Python's io module and the :func:`mne.io.read_raw_edf` function. Note
 # that we must use `preload=False`, the default in MNE-Python. It prevents the
 # data from being loaded and modified when converting to BIDS.
+
+# Load the data from "2 minutes eyes closed rest"
 edf_path = eegbci.load_data(subject=1, runs=2)[0]
 raw = mne.io.read_raw_edf(edf_path, preload=False)
 
-###############################################################################
-# The annotations stored in the file must be read in separately and converted
-# into a 2D numpy array of events that is compatible with MNE.
-annot = mne.read_annotations(edf_path)
-raw.set_annotations(annot)
+# For converting the data to BIDS, we need to convert the the annotations
+# stored in the file to a 2D numpy array of events.
 events, event_id = mne.events_from_annotations(raw)
 
-print(raw)
+###############################################################################
+# For the sake of the example we will also pretend that we have the electrode
+# coordinates for the data recordings.
+# We will use a coordinates file from the MNE testing data in `CapTrak`_
+# format.
+#
+# .. note:: The ``*electrodes.tsv`` and ``*coordsystem.json`` files in BIDS are
+#           intended to carry information about digitized (i.e., *measured*)
+#           electrode positions on the scalp of the research subject. Do *not*
+#           (!) use these files to store "template" or "idealized" electrode
+#           positions, like those that can be obtained from
+#           :func:`mne.channels.make_standard_montage`!
```

Comment:
I still don't understand why :) Is it because BIDS demands this? Wouldn't it make sense to add a tag to those files, indicating whether the coordinates were measured or are templates? (This is beyond the scope of this PR, obviously; I'm just curious.)

Reply:
Because the thing is, even if you _do_ use template coordinates, there are so many different templates and caps out there… it can make total sense to supply those with the dataset!

Reply:
> I still don't understand why

because just like `fsaverage` is shipped with freesurfer, template coordinate positions should be (and are) shipped with the analysis softwares (like MNE). --> it doesn't make sense to duplicate one and the same template a million times :shrug: 

> Is it because BIDS demands this?

it's currently unclear and apart from me and perhaps Robert O., there are apparently very few people interested in this (for BIDS at least) https://github.com/bids-standard/bids-specification/issues/318

> Wouldn't it make sense to add a tag to those files

an additional field in `coordsystem.json` could make sense, yes ... and that'd be preferable to a wild and unknown mixture of template and measured coordinates

>  it can make total sense to supply those with the dataset!

I don't think so, I think that if you use a special template, you should make a PR to the analysis software of your choice